test/scripts/deploy-ramen

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import os
+import tempfile
+import concurrent.futures
+from subprocess import run
+
+TOP_DIR = os.path.dirname(os.path.dirname(os.path.dirname(__file__)))
+IMAGE = "quay.io/ramendr/ramen-operator:latest"
+HUB = "hub"
+MANAGED_CLUSTERS = ["dr1", "dr2"]
+
+with tempfile.TemporaryDirectory(prefix="ramen-test-") as tmpdir:
+    tar = os.path.join(tmpdir, "image.tar")
+    print(f"Saving {IMAGE} to {tar}")
+    run(["podman", "save", IMAGE, "-o", tar], check=True)
+
+    def load_image(cluster):
+        print(f"Loading image in cluster '{cluster}'")
+        cmd = ["minikube", "--profile", cluster, "image", "load", tar]
+        run(cmd, check=True)
+
+    clusters = [HUB] + MANAGED_CLUSTERS
+    with concurrent.futures.ThreadPoolExecutor() as executor:
+        list(executor.map(load_image, clusters))
```

Comment:
Would be nice (unless it is already automatic) to delete the tmpdir as over time these may accumulate. Potentially these are in `/tmp` which should get cleaned up on a reboot.

Reply:
Yes the temporary directory is removed when we exit the `with` block:
https://docs.python.org/3.6/library/tempfile.html#tempfile.TemporaryDirectory